crates/cheatcodes/src/fs.rs

```diff
@@ -382,7 +382,11 @@ mod tests {
             root: PathBuf::from(&env!("CARGO_MANIFEST_DIR")),
             ..Default::default()
         };
-        Cheatcodes { config: Arc::new(config), ..Default::default() }
+
+        let mut cheatcodes = Cheatcodes::default();
+        cheatcodes.config = Arc::new(config);
+
+        cheatcodes
```

Comment:
Probably an unintended change.

Reply:
The persistent_deps is a private field so we can't use the previous syntax anymore

Reply:
Let's just make `persistent_deps` also `pub`, that's probably more consistent with other fields in the `Cheatcodes` that are also `pub`.